### ISSUE TITLE
`svpack filter --min-svlen` removes all BND records

### ISSUE THREAD

**Issue by williamrowell** [amwenger/svpack#9]
`svpack filter --min-svlen` removes all BNDs.  I wouldn't expect this to be the default behavior, and I don't believe this was always the behavior.

```bash
❯ zgrep -v -c '^#' HG002.GRCh38.pbsv.BND_only.vcf.gz
90

❯ python svpack/svpack filter --pass-only HG002.GRCh38.pbsv.BND_only.vcf.gz | grep -v -c '^#'
52

❯ python svpack/svpack filter --pass-only HG002.GRCh38.pbsv.BND_only.vcf.gz --min-svlen 0 | grep -v -c '^#'
52

❯ python svpack/svpack filter --pass-only HG002.GRCh38.pbsv.BND_only.vcf.gz --min-svlen 50 | grep -v -c '^#'
0
```

I'd expect BND SVLEN to be either `INFO/MATEDIST` or `+inf`.

**Comment by Gedofs** [comment]
thanks

**Comment by williamrowell** [comment]
@Gedofs: Might take a few days for the fix to get merged, but you can download my patched version here: https://github.com/williamrowell/svpack/blob/bnd_svlen/svpack

**Comment by amwenger** [comment]
Thank you, @williamrowell.  This is a regression that was introduced in 2876819 trying to add support for callers that do not use SVLEN.

I like your fix in PR #10.  Thank you for submitting with a fix! I will merge it and then make one modification: setting SVLEN to `None` instead of `math.inf` when the BND is interchromosomal.  Using `math.inf` would require additional logic in `svpack match`.

**Comment by amwenger** [comment]
Fixed by PR #10 and updated in d9935ca.
